### PR TITLE
mesa: update to 20.1.4, fix patches

### PR DESCRIPTION
**Pull request by BouncyMaster**
`fix-libXvMC-versioning.patch` was fixed upstream [here](https://github.com/mesa3d/mesa/commit/a35c4675a72c30e6b7e116237a43ca7c520b060d).
For `clover-ppc64-cpp14.patch`, the `clover_cpp_std` variable was removed completely, so not sure if that still works.

### CODE REVIEW

File: srcpkgs/mesa/template [hunk 1]
```diff
@@ -23,7 +23,7 @@ license="MIT, LGPL-2.1-or-later"
 homepage="https://www.mesa3d.org/"
 changelog="https://www.mesa3d.org/relnotes/${version}.html"
```
Comment (travankor) [inline srcpkgs/mesa/template:24]: https://docs.mesa3d.org/relnotes/20.1.3.html

**Comment by fosslinux** [comment]
Probably should squash the commits.

**Comment by BouncyMaster** [comment]
Done

**Comment by st3r4g** [comment]
.4 is out

**Comment by BouncyMaster** [comment]
> .4 is out

Will update this PR

**Comment by fosslinux** [comment]
>  For clover-ppc64-cpp14.patch, the clover_cpp_std variable was removed completely, so not sure if that still works.

Pinging @q66

**Comment by fosslinux** [comment]
>  For clover-ppc64-cpp14.patch, the clover_cpp_std variable was removed completely, so not sure if that still works.

Pinging @q66

**Comment by BouncyMaster** [comment]
Removed patch `parallel-build.patch`, possibly fixed upstream [here](https://github.com/mesa3d/mesa/commit/b3fe8cb8d9688082eadf8b5e25f0d4990f825193). Not sure though, need @q66 to look at this.

**Pull request merged**